### PR TITLE
Update dependencies, fix hub API calls, new Google icon

### LINKED ISSUE #68: Update Google Developers icon to new version

### PR DESCRIPTION
**Pull request by Splaktar**
update angular-material and angularjs.
use https to access the hub APIs.
styling updates after angular-material 0.11.3 update
use new Google Developers icon.
Don't round image bottom corners on cards that have content below the image.

Closes #68.

### CODE REVIEW

**Comment by Splaktar** [comment]
screen shot 2015-10-18 at 1 23 31 pm
screen shot 2015-10-18 at 1 22 15 pm
screen shot 2015-10-18 at 1 21 56 pm

File: app/about/about.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h2 class="md-toolbar-tools">
         <md-card>
             <md-card-content>
             <a href="https://developers.google.com/">
-                <img class="sponsor" src="/app/images/Google-Developers.png" alt="Google Developers"
+                <img class="sponsor" src="/app/images/logo_lockup_developers_icon_vertical.png" alt="Google Developers"
```
Comment (stefanhoth) [inline app/about/about.html]: I know it's nitpick (so forgive me) but could you rename the file to make it a bit easier to recognize it as the Google Developers logo? Or is there a benefit of using "Google's" naming?

File: app/events/events.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ angular.module('gdgXBoomerang')
     vm.dateFormat = Config.dateFormat;
     vm.events = { past:[], future:[] };
 
-    var url = 'http://hub.gdgx.io/api/v1/chapters/' + Config.id + '/events/upcoming?callback=JSON_CALLBACK';
+    var url = 'https://hub.gdgx.io/api/v1/chapters/' + Config.id + '/events/upcoming?callback=JSON_CALLBACK';
```
Comment (stefanhoth) [inline app/events/events.js]: :100:

**Comment by stefanhoth** [comment]
Thanks, that looks good!

I'm wondering if there will be ever something better than this when dealing with version updates (updating them in several files for different reasons looks like a waste).

File: app/about/about.html (hunk 1 above)
Comment (Splaktar) [inline app/about/about.html]: Done

**Comment by Splaktar** [comment]
Currently just need to update it in `CDN.json` and `bower.json`. The `index.html` is updated by the `gulp prod` build. We'll probably move to just using NPM instead of Bower at some point, but that won't really make things too much better. The reason that the `CDN.json` is needed is due to projects not putting the proper metadata in their `bower.json` files, maybe with only using npm this will improve.

**Comment by stefanhoth** [comment]
Thanks for the explanation. LGTM :+1: but I'll wait for a few hours to let someone else take a look.

**Pull request merged**